src/index.tsx

```diff
@@ -2,6 +2,10 @@ import { createRoot } from 'react-dom/client'
 import 'typeface-roboto'
 
 import { App } from './App'
+import { randomNumber } from './services/RandomNumber'
+
+const seed = new URLSearchParams(window.location.search).get('seed')
+randomNumber.init(seed ?? undefined)
```

Comment:
Assuming we replace `init` with a `RandomNumberService` constructor, can this logic be moved to where the `randomNumber` singleton is initialized in `src/services/RandomNumber/index.ts`?